### PR TITLE
[MDS-5400] Fix bug on radio button

### PR DESCRIPTION
**Pull request by henryoforeh-dev**
## Objective 

[MDS-5400](https://bcmines.atlassian.net/browse/MDS-5400)

Fixed bug on radio button. The bug occurs when trying to save a strung data into a boolean column in the database.

### CODE REVIEW

**Review by matbusby-fw — APPROVED** [review]
Nice!

File: services/core-web/src/components/common/RenderRadioButtons.js [hunk 1]
```diff
@@ -23,13 +23,16 @@ const defaultProps = {
 
 const RenderRadioButtons = (props) => {
   const { meta, label, disabled, input, id, customOptions } = props;
-  const radioValue = typeof input.value !== "undefined" ? input.value.toString() : "";
 
   const options = customOptions ?? [
-    { label: "Yes", value: "true" },
-    { label: "No", value: "false" },
+    { label: "Yes", value: true },
+    { label: "No", value: false },
```
Comment (simensma-fresh) [inline services/core-web/src/components/common/RenderRadioButtons.js:29]: Will this affect any of our backend endpoints that expect a boolean as a string or would this purely be a frontend thing? Looks like the be checks for booleans as a string in a couple of places:
image

Comment (henryoforeh-dev) [reply services/core-web/src/components/common/RenderRadioButtons.js:29]: Good question, I will check.

Comment (henryoforeh-dev) [reply services/core-web/src/components/common/RenderRadioButtons.js:29]: I think it should be fine. The examples in the screenshot are for checks that happen during filtering. The respective columns in the tables in the backend store the data as boolean.
 
I also saw another ideal example in the explosives permit form. **closedOptions** is passed as an argument to customOptions parameter with boolean option values. 

**
image
**

image

**Review by simensma-fresh — APPROVED** [review]

**Pull request merged**